testpar/t_init_term.c

```diff
@@ -37,11 +37,11 @@ main(int argc, char **argv)
 
     MPI_Finalize();
 
-    nerrors += GetTestNumErrs();
-
     /* test if we can initialize the library with MPI being finalized
        and create a file serially */
-    H5open();
+    if (H5open() < 0) {
```

Comment:
We should use the VRFY macro for these checks since it's consistent and will increment nerrors on failure